utility.py

```diff
@@ -0,0 +1,95 @@
+import os
+import fnmatch
+import zipfile
+import tempfile, shutil
+
+
+def get_input():
+    """
+
+    :return:
+    """
+    file_path = input('Enter the zip file path')
+    # C:\Users\gaura\OneDrive\Desktop\Grainger\NewTestImages.zip
+    # C:\Users\gaura\OneDrive\Desktop\Grainger\NewTestImages\TestImages.zip
+    num_rows = int(input('Enter number of rows'))
+    num_col = int(input('Enter number of columns'))
+    return file_path, num_rows, num_col
+
+
+def validate_input(file_path, num_rows, num_columns):
+    """
+
+    :param num_images:
+    :param num_rows:
+    :param num_columns:
+    :return:
+    """
+    session_dir = tempfile.mkdtemp(dir='static')
+    print(session_dir)
+
+    extract_files(file_path, session_dir)
+
+    image_extension = get_file_extension(session_dir)
+
+    num_images = get_file_count(image_extension, session_dir)
+    print('num_images', num_images)
+    return num_columns * num_rows == num_images
+
+
+def extract_files(file_path, full_session_path):
+    with zipfile.ZipFile(file_path, "r") as zip_ref:
+        print('list of dir', zip_ref.namelist())
+
+        for member in zip_ref.namelist():
+            filename = os.path.basename(member)
+            # skip directories
+            if not filename:
+                continue
+
+            # copy file (taken from zipfile's extract)
+            print('member=',member)
```

Comment:
remove unnecessary print statements

Reply:
@wallacerob I have incorporated the changes. Please validated